distribution/docs/src/main/resources/content/_core-concepts/data-standards.adoc

```diff
@@ -0,0 +1,39 @@
+:type: subCoreConcept
+:status: published
+:title: Data Standards added by ${cal-branding}
+:parent: Standards Supported by ${branding}
+:order: 011
+
+.{title}
+[cols="2,1m,1" options="header"]
+|===
+
+|Format
+|File Extensions
+|Additional Metadata Attributes Available (if populated)
+
+|http://www.gwg.nga.mil/ntb/baseline/docs/2500c/index.html[NITF 2.0 and 2.1]
+|ntf
+|<<_file_formats_supported,Standard attributes>> and<<_nitf_attribute_mappings,NITF Attribute Mappings>>
```

Comment:
Should there be a space after *and* here and below?

Reply:
fixed.